### ISSUE TITLE
Add way to configure object complexity?

### ISSUE THREAD

**Issue by yamalight** [davidwaterston/eslint-plugin-objects#1]
Would it be possible to have a way to say that I want the rules to apply only to objects that has more than X properties?
E.g. I want to allow using things like this: 

``` js
doThings({just: '123', twoProps: 'value'});
```

But disallow writing complex objects (more than 3 props or have nested objects, etc).

**Comment by davidwaterston** [comment]
@yamalight Yes, that should definitely be possible and would make a good enhancement.
I'll look into implementing that this weekend.

My first thoughts are that I'll just add the ability to pass a second parameter in the config file that indicates when the rule should be applied (e.g. the number of properties that must exist before the rule applies). So, for 'no-object-properties-one-line', something like this:

```
{
    "rules": {
        "objects/no-object-properties-one-line": [2,3]
    }
}
```

In this case, the maximum number of properties allowed on one line would be 3 (the second param) - if more were placed on a single line then an error would be thrown.
If I default the 'limit' to 1 then the rules would function as they do now.

Let me know if you have different ideas about how this should be implemented...

**Comment by yamalight** [comment]
Maybe it'd be better to use object config similar to [key-spacing](http://eslint.org/docs/rules/key-spacing) rule. Then it'd be possible to extend it later without breaking compatibility. 
E.g:

``` json
{
    "rules": {
        "objects/no-object-properties-one-line": [2, {"properties": 3, "nested": 1}]
    }
}
```

What do you think?

**Comment by davidwaterston** [comment]
Yes, that looks better... I'll take a look at how key-spacing implements that.
Thanks for the suggestion.

**Comment by yamalight** [comment]
Cool, thanks for quick reply!
